cmd/protoc-gen-cpp-tableau-loader/embed/hub.pc.h

```diff
@@ -142,7 +149,7 @@ class Messager {
 
 class Hub {
  public:
-  Hub() = default;
+  Hub(const HubOptions* options = nullptr) : options_(options ? *options : HubOptions{}) {}
   Hub(MessagerContainer container) { SetMessagerContainer(container); }
```

Comment:
- Also should change another constructor: `Hub(MessagerContainer container, const HubOptions* options = nullptr)`
- And check where to use this constructor.